pkgs/development/tools/misc/ccache/default.nix

```diff
@@ -15,43 +15,46 @@ let ccache = stdenv.mkDerivation rec {
     substituteInPlace Makefile.in --replace 'objs) $(extra_libs)' 'objs)'
   '';
 
-  doCheck = true;
+  doCheck = !stdenv.isDarwin;
```

Comment:
This doesn't affect anything as tests are run and do output in the build directory. Also I've tried running the tests without nix against CLT and they are also broken:
```
CC='gcc' ./test.sh
gcc ---------------
WARNING: Compiler gcc not supported (version: Configured with: --prefix=/Library/Developer/CommandLineTools/usr --with-gxx-include-dir=/usr/include/c++/4.2.1) -- not running tests
```
If I patch things to actually run the test:
```
SUITE: "base", TEST: "-D not hashed" - Expected "cache hit (preprocessed)" to be 14, got 13
cache directory                     /tmp/ccache-3.2.5/testdir.95046/.ccache
primary config                      /tmp/ccache-3.2.5/testdir.95046/.ccache/ccache.conf
secondary config      (readonly)    /usr/local/etc/ccache.conf
cache hit (direct)                     0
cache hit (preprocessed)              13
cache miss                            40
called for link                        2
called for preprocessing               2
multiple source files                  1
compiler produced stdout               1
couldn't find the compiler             1
bad compiler arguments                 1
unsupported source language            2
unsupported compiler option            1
output to a non-regular file           1
no input file                          1
files in cache                         3
cache size                          12.3 kB
max cache size                       5.0 GB
TEST FAILED
```